src/whip-client.c

```diff
@@ -321,6 +322,27 @@ static void whip_options(void) {
 	WHIP_LOG(LOG_INFO, "\n");
 }
 
+static gboolean source_events(GstPad* pad,
+														GstObject* parent,
+														GstEvent* event) {
```

Comment:
Please put this in a single line: there's not many arguments, so they fit.
As a side note, per code style the `*` needs to be next to the pointer, not the type, so it should be `GstPad *pad` and not `GstPad* pad` for instance. Same for the other arguments and a few other pointers you've declared in your additions.